torch/testing/_internal/common_methods_invocations.py

```diff
@@ -17500,6 +17520,8 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
         # and scatter_reduce hasn't been added to the whitelist in gen_variable_type yet
         dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
         dtypesIfCUDA=floating_types_and(torch.float16, torch.bfloat16),
+        # FIXME: vmap segfaults in _remove_batch_dim after node logic was added to scatter_reduce_backward(prod)
```

Comment:
Ouch
cc @zou3519

Reply:
This segfault might have been due to a print statement I had added somewhere locally, my bad. I don't think there was a segfault otherwise, will remove this.

Reply:
My bad, this was a mistake on my part likely introduced by a print statement.